newdle/client/src/components/App.js

```diff
@@ -37,6 +37,7 @@ export default function App() {
             <Route exact path="/mine" component={MyNewdles} />
             <Route exact path="/participating" component={NewdlesParticipating} />
             <Route path="/newdle/:code/summary" component={SummaryPage} />
```

Comment:
unrelated to your changes, but this one should probably be `exact` since the summary doesn't have any sub-urls, does it?

Reply:
It doesn't, I usually omit it if there are no conflicts. But, we should remain consistent.